core/server/api/v2/posts.js

```diff
@@ -0,0 +1,179 @@
+const models = require('../../models');
+const common = require('../../lib/common');
+const urlService = require('../../services/url');
+const allowedIncludes = ['created_by', 'updated_by', 'published_by', 'author', 'tags', 'authors', 'authors.roles'];
+const unsafeAttrs = ['author_id', 'status', 'authors'];
+
+module.exports = {
+    docName: 'posts',
+    browse: {
+        options: [
+            'include',
```

Comment:
Was the move away from usage of `browseDefaultOptions` intentional? For this specific instance, it used to be `browseDefaultOptions` concated with `extraOptions`. Just qurious about the thinking here :thinking:

Reply:
A shared options array is currently not supported by the frame.